vignettes/getting_started.Rmd

```diff
@@ -17,20 +17,21 @@ knitr::opts_chunk$set(
 
 This vignette provides a quick tour of the `osfr` package.
 
-[OSF](https://osf.io), developed by the [Center for Open Science (COS)](https://cos.io/), is a free, open source, web application that provides a space for researchers to collaborative store, manage, and share their research materials (e.g. data, code/analysis scripts, protocols).Files can be kept private, shared with a specific group of researchers, or made publically available with citable DOIs so researchers can get credit for their work.
+[OSF](https://osf.io), developed by the [Center for Open Science (COS)](https://cos.io/), is a free, open source, web application that provides a space for researchers to collaboratively store, manage, and share their research materials (e.g. data, code, protocols). Files can be kept private, shared with a specific group of researchers, or made publicly available with citable DOIs so researchers can get credit for their work.
 
-Currently, the `osfr` is allows researchers to create and modify project, components, and files through the OSF API.
+`osfr` allows researchers to create and modify projects, components, and files through OSF's API.
 
 ## Working with projects/components
 
-Most work on the OSF is organized around `projects`. `Projects` can `files`, groups of files in `folders`, and/or files in subsections of a project called `components`. Collectively `projects` and `components` are called `nodes`. 
+Most work on OSF is organized around ***projects***. Projects can contain *files*, groups of files in *directories*, and/or files in subsections of a project called ***components***. Collectively projects and components are called ***nodes***.
```

Comment:
Good call. I've been using directory instead of folder to be consistent with `osf_mkdir()`. I also like that directory and file starts with a different letter. I've been considering adding some kind of visual type indicator to `osf_tbls` to distinguish between files, directories, projects, components, and users—and I think we can leverage the fact that they all start with different letters.